src/lib/runtimeLoader.ts

```diff
@@ -1,6 +1,11 @@
-module.exports = function(content) {
+module.exports = function(content: string) {
+  // @ts-ignore
   this.cacheable && this.cacheable();
+
+  // @ts-ignore
   const params = JSON.parse(this.query.slice(1));
```

Comment:
And defining new properties `cacheable` and `query` on globalThis is not worth for this specific use case, so ignoring this problem.